### PR TITLE
review-dev_guide_for_online

### PR DESCRIPTION
**Pull request by sallyom**
WIP 
@ahardin-rh I'm going to have @Miciah (if you will please) look this over..

### CODE REVIEW

File: dev_guide/routes.adoc [hunk 1]
```diff
@@ -60,7 +61,35 @@ spec:
...
+.YAML Definition of the Unsecured Route Created Above
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: frontend
+spec:
+  host: frontend-projectname.{product-title} {product-version} hostname
```
Comment (Miciah) [inline dev_guide/routes.adoc]: What's up with `.{product-title} {product-version} hostname`? Anyway, `host` should not be specified at all.

File: dev_guide/routes.adoc [hunk 2]
```diff
@@ -60,7 +61,35 @@ spec:
     name: frontend
 ----
 ====
+endif::[]
+ifdef::openshift-online[]
```
Comment (Miciah) [inline dev_guide/routes.adoc]: Could we used a finer-grained ifdef? It's just the command and yaml that need to be conditionalised.

File: dev_guide/routes.adoc [hunk 3]
```diff
@@ -121,7 +150,30 @@ remove a passphrase from a keyfile, you can run:
...
+====
+link:https://en.wikipedia.org/wiki/Transport_Layer_Security[TLS] is the
+replacement of SSL for HTTPS and other encrypted protocols.
+====
+
+----
+$ oc create route edge --service=frontend
+----
+
+====
```
Comment (Miciah) [inline dev_guide/routes.adoc]: Might as well include yaml for an edge route for Online too.

**Review by Miciah — CHANGES_REQUESTED** [review]

File: dev_guide/routes.adoc (hunk 1 above)
Comment (ahardin-rh) [inline dev_guide/routes.adoc]: In code blocks, {product-title} and {product-version} will not render properly

**Comment by sallyom** [comment]
@Miciah, I've followed your suggestions here, mind checking it again? Thanks.

File: dev_guide/routes.adoc [hunk 4]
```diff
@@ -121,7 +148,47 @@ remove a passphrase from a keyfile, you can run:
...
+    name: frontend
+  tls:
+    termination: edge
+----
+====
+
+====
+endif::[]
 
+ifndef::openshift-online[]
```
Comment (Miciah) [inline dev_guide/routes.adoc]: Would it be clearer to include this with the `ifndef::openshift-online[]` block again, without the intervening `ifdef::openshift-online[]` block?

File: dev_guide/routes.adoc [hunk 5]
```diff
@@ -121,7 +148,47 @@ remove a passphrase from a keyfile, you can run:
...
 ----
+endif::[]
+ifdef::openshift-online[]
+Unsecured routes are the default configuration, and are therefore the simplest
+to set up.
+However,
+xref:../architecture/core_concepts/routes.adoc#secured-routes[secured routes]
+offer security for connections to remain private.  To create a secured HTTPS route
+encrypted with the default certificate for {product-title} {product-version}
+you can use the `create route` command.
```
Comment (Miciah) [inline dev_guide/routes.adoc]: Only the last sentence of the paragraph varies between Online and non-Online, so the ifdeffing could be still a little more fine-grained, but I don't feel strongly about it.

**Review by Miciah — APPROVED** [review]
A couple modest suggestions, but looks fine.

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**